app/src/main/java/com/io/movio/ui/movielist/MovieListViewModel.kt

```diff
@@ -1,18 +1,17 @@
-package com.io.movio.viewmodels
+package com.io.movio.ui.movielist
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import com.io.movio.models.Movie
-import com.io.movio.repositories.MovieRepository
+import com.io.movio.data.models.Movie
+import com.io.movio.data.repositories.MovieRepository
+import com.io.movio.domain.GetMoviesUseCase
 
 class MovieListViewModel : ViewModel() {
-
     private val _movieList = MutableLiveData<List<Movie>>()
     val movieList: LiveData<List<Movie>> = _movieList
 
     init {
-        _movieList.value = MovieRepository.getMovies()
-    }
-
-}
\ No newline at end of file
+        _movieList.value = GetMoviesUseCase(MovieRepository).getMovies()
```

Comment:
if you remove the Repository from the constructor you can then just call GetMoviesUseCase() and because of the invoke operator this will call getMovies internally